src/gwt/src/org/rstudio/core/client/ElementIds.java

```diff
@@ -179,6 +179,13 @@ public static String idFromLabel(String label)
    public final static String TERMINAL_GENERAL_PREFS = "terminal_general_prefs";
    public final static String TERMINAL_CLOSING_PREFS = "terminal_closing_prefs";
 
+   public final static String NEW_PROJECT_DIRECTORY = "new_project_directory";
+   public final static String NEW_PROJECT_TYPE = "new_project_type";
+   public final static String NEW_PROJECT_GIT_REPO = "new_project_git_repo";
+   public final static String NEW_PROJECT_RENV = "new_project_renv";
+   public final static String NEW_PROJECT_SOURCE_FILES = "new_project_source_files";
+   public final static String NEW_PROJECT_NEW_SESSION = "new_project_new_session";
+
```

Comment:
Haven't looked to see why, so sticking comment here.

## Problem
duplicate elementIDs in New Project Wizard after building this PR
## Repro
File / New Project, then run accessibility test scoped just to the dialog's top element (I'm using Axe devtools pro which is, unfortunately, a paid product now; apparently they want to stay in business)
## Details
Before this change there was already a problem with duplicate IDs, not previously noticed by me. Specifically:
* Document has multiple static elements with the same id attribute: `rstudio_label_website_using_blogdown_wizard_page`
* Document has multiple static elements with the same id attribute: `rstudio_label_simple_r_markdown_website_wizard_page`

Up to you if you want to fix those as part of this, certainly not essential, these are unlikely to be used for automation. I will do another broader sweep for duplicate IDs soon, and have a long-standing TODO to implement testing for it in the Selenium automation.

New problems from this PR:
* Document has multiple elements referenced with ARIA with the same id attribute: `rstudio_new_project_directory`
* Document has multiple static elements with the same id attribute: `rstudio_new_project_git_repo` (15 dups!!)

Reply:
Apparently we create the whole wizard dialog tree up front instead of showing one at a time. I'll take a look.